User-Help/ModeratorFAQ.md

```diff
@@ -0,0 +1,30 @@
+# So you're a moderator now
+So you've been given moderator status on a Codidact site. Welcome! This is what that means for you.
+
+## Your community is your community
+Codidact sites are community-run. Codidact is here to support you and your community, but you make the decisions about how you run things yourselves. We'll provide our advice and guidance and let you know what we think works well, but if your community wants to go in a different direction, that's up to you!
+
+All of our communities must abide by the Codidact Code of Conduct, which is [in your help center](/policy/code-of-conduct).
+
+## You're a community leader
+Particularly in the early stages of your site, you'll be one of the most prominent figures. Your actions will help to build and shape your site and your community, and set the standards and norms for those who come after you.
+
+One of the most important things you can do for your community is to help guide discussion. Remember that being a moderator doesn't make you More Right™, but it does mean that your words carry a certain weight and your community looks to you as a guiding figure. Offer your experience and advice, but avoid throwing your weight around and using your position to shut down people who disagree with you.
+
+This also means that your actions can set the tone for the community - if you're welcoming and inclusive and you encourage others to do likewise, your community will be welcoming and inclusive too.
+
+### We'll listen to you
+Codidact staff keep an eye on all communities, but we don't always see every post. If there's something you think needs our attention or something your community needs that we can help with, let someone know - you can email us at info@codidact.org, or we'll often respond more quickly in Discord (see below). As a leader in your community, you know your community better than we do, so we'll defer to you as much as possible.
```

Comment:
They can also flag if that makes sense in context.  Just make it clear that the flag is directed to admins, else we'll generally leave it for them to handle.

Reply:
I was a little wary of that because that implies that we'll monitor flag queues, and while that's probably true at the moment, we may not always want to do that.

Reply:
Oh, that's a good point.  Yeah, let's not commit to that.  This is fine.